docs/source/algorithms/Stitch1D-v3.rst

```diff
@@ -10,27 +10,62 @@ Description
+
+Either the right-hand-side or left-hand-side workspace can be chosen to be scaled.
+Users can optionally provide :ref:`algm-Rebin` :literal:`Params`, otherwise they are calculated from the input workspaces.
+Likewise, :literal:`StartOverlap` and :literal:`EndOverlap` are optional. If not provided, then these
+are taken to be the region of X-axis intersection.
+
+The algorithm workflow is as follows:
+
+#. The workspaces are initially rebinned, as prescribed by the rebin :literal:`Params`. Note that
+   rebin parameters are determined automatically if not provided. In this case, the step size is
+   taken from the step size of the LHS workspace (or from the RHS workspace if :literal:`ScaleRHSWorkspace`
+   was set to false) and rebin boundaries are taken from the minimum X value in the LHS workspace
+   and the maximum X value in the RHS workspace respectively.
+#. After rebinning, each spectrum is searched for special values. Special values refer to signal
+   (Y) and error (E) values that are either infinite or NaN. These special values are masked out
+   as zeroes and their positions are recorded for later reinsertion.
+#. Next, if :literal:`UseManualScaleFactor` was set to false, both workspaces will be integrated
+   according to the integration range defined by :literal:`StartOverlap` and :literal:`EndOverlap`.
+   Note that the integration is performed without special values, as those have been masked out
+   in the previous step. The scale factor is then calculated as the quotient of the integral of
+   the left-hand-side workspace by the integral of the right-hand-side workspace (or the quotient
+   of the right-hand-side workspace by the left-hand-side workspace if :literal:`ScaleRHSWorkspace`
+   was set to false), and the right-hand-side workspace (left-hand-side if :literal:`ScaleRHSWorkspace`
+   was set to false) is multiplied by the calculated factor.
+#. Alternatively, if :literal:`UseManualScaleFactor` was set to true, the scale factor is applied
+   to the right-hand-side workspace (left-hand-side workspace if :literal:`ScaleRHSWorkspace` was
+   set to false).
+#. The weighted mean of the two workspaces in range [:literal:`StartOverlap`, :literal:`EndOverlap`]
+   is calculated. Note that if both workspaces have zero errors, an un-weighted mean will be
+   performed instead.
+#. The output workspace will be created by summing the left-hand-side workspace (values in range
+   [:literal:`StartX`, :literal:`StartOverlap`], where :literal:`StartX` is the minimum X value
+   specified via :literal:`Params` or calculated from the left-hand-side workspace) + weighted
+   mean workspace + right-hand-side workspace (values in range [:literal:`EndOverlap`, :literal:`EndX`],
+   where :literal:`EndX` is the maximum X value specified via :literal:`Params` or calculated
+   from the right-hand-side workspace) multiplied by the scale factor.
+#. The special values are put back in the output workspace.
```

Comment:
I think scientists are more worried about how errors are propagated, as they were not sure that they are propagated correctly. For instance, they didn't know if the scale factor (when calculated by the algorithm) had an error associated. From the code, I think it does have an error associated and this error is propagated correctly.